src/datachain/lib/diff.py

```diff
@@ -16,6 +17,21 @@
 C = Column
 
 
+def get_status_col_name() -> str:
```

Comment:
Does the column name need to be random? Can we have a default column name that can be changed by users?

Eg:
```
def compare(col="status"):
   pass


dc.compare(col=...)
```